xml/deploy_cephadm.xml

```diff
@@ -1193,6 +1193,21 @@ config: OK
      deployment will use the intended cluster network from the start.
     </para>
    </note>
+   <para>
+     If your cluster is set to have dense nodes (greater than 62 OSDs per host),
+     make sure to assign sufficient ports for &ceph; OSDs. The default (6800-7300)
```

Comment:
I am not sure it is, but if this is a _range_ of ports, then I think "the default range (6800-7300)" would be preferable.

Reply:
Good question, I presume it's a range based on how we've phrased this here: https://susedoc.github.io/doc-ses/master/html/ses-troubleshooting/storage-tips.html#storage-bp-net-firewall

@jan--f ?

Reply:
Yep makes sense!